prototype/water-extent-map-vertex.ipynb

```diff
@@ -0,0 +1,124 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import copy\n",
+    "\n",
+    "import ipywidgets as widgets\n",
+    "from IPython.display import display\n",
+    "\n",
+    "import hyp3_sdk"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hyp3_watermap = hyp3_sdk.HyP3('https://hyp3-watermap.asf.alaska.edu', prompt=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "granules = widgets.Textarea(\n",
+    "    placeholder='copy-paste Sentinel-1 granule names or File IDs here (One granule per line)',\n",
+    "    layout=widgets.Layout(width='100%', height='12em'),\n",
+    ")\n",
+    "display(granules)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all_granules = [granule.strip().split('-')[0] for granule in granules.value.splitlines()]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_definition = {\n",
+    "    'name': 'water-map-example',\n",
```

Comment:
could consider separate names for the sdk vs vertex notebooks, e.g. `'name': 'water-map-example-sdk'`

Reply:
that's 21 characters...

Reply:
I combined the two notebooks to reduce duplication, so this should be resolved